shared/gh/js/bootstrap.admin-listview.js

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright 2014 Digital Services, University of Cambridge Licensed
+ * under the Educational Community License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+define(['gh.core'], function(gh) {
+
+    var modules = null;
+
+
+    /////////////
+    // BINDING //
+    /////////////
+
+    $('body').on('click', '.gh-series-select', function() {
+        // Remove the active state from other series
+        $('.gh-series-select').removeClass('gh-series-active');
+        // Add the active state to the clicked series
+        $(this).addClass('gh-series-active');
+
+        // TODO: Load the series in batch edit mode
+    });
+
```

Comment:
Because I can't find anything else to comment on; this empty line can probably go :")